src/storage/mongodb/SiteAreaStorage.ts

```diff
@@ -280,6 +280,8 @@ export default class SiteAreaStorage {
         tenantID: tenant.id, aggregation, localField: '_id', foreignField: 'siteAreaID',
         asField: 'chargingStations', pipelineMatch
       }, additionalPipeline);
+      // Disable
+      params.withChargingStations = false;
```

Comment:
This is crashing the smart charging functionality, because on line 438 all the charging stations are deleted from the site area, if the parameter is set to false.